app/views/flash_messages/_flash_messages.html.erb

```diff
@@ -0,0 +1,7 @@
+<% # loop through and render any flash messages %>
+
+<% flash.each do | type, message | %>
+  <% if type != 'title' %>
```

Comment:
Totally, the flash messages have come from a Rails Bootcamp tutorial I did in September, so that's why it's a little old skool - plus I keep forgetting about `unless` :) Thanks for this!